openbas-api/src/main/java/io/openbas/rest/helper/RestBehavior.java

```diff
@@ -133,6 +158,13 @@ public ViolationErrorBag handleIntegrityException(DataIntegrityViolationExceptio
   }
 
   @ExceptionHandler(ElementNotFoundException.class)
+  @ApiResponses(
+      value = {
+        @ApiResponse(
+            responseCode = "404",
+            description = "Resource not found",
+            content = @Content(schema = @Schema(implementation = ValidationErrorBag.class))),
```

Comment:
not sure if the schema implementation should be a ValidationErrorBag as we are returning a response entity instead

Reply:
Bad copy paste, fixing :)